conf/config_utils.py

```diff
@@ -7,16 +7,12 @@
 
 
 def from_file(fname):
-    if not os.path.exists(fname):
-        log.warning('config file does not exist: "{}"'.format(fname))
-        return {}
     try:
         with open(fname) as buf:
             conf = json.load(buf)
             return conf
-    except Exception:
-        log.warning('Error loading config file', exc_info=1)
```

Comment:
I said a stupid thing. You *removed* that exception handling. Instead, you only catch `EnvironmentError`, which is what happens when, for example, you don't have permissions.

You should add an `except ValueError` clause